src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString(){
+        return value;
+    }
```

Comment:
You should be appending the "Address: " here.

Similar for the method implemented in other classes.